coralproject/talk
# Email Attach Fixes

## What does this PR do?

- Fixes bug in `excludeIf` invocation to include `inProgress` flag to allow wizard to complete
- Wrapped `Popup` `onunload` inspection with `try {} catch {}` to protect against security exemption errors throwing off the logic
- Prevent _Attach Email Dialog_ from showing when there is already the _Set Username Dialog_

## How do I test this PR?

1. Create a new account with an external auth provider (google/facebook/etc)
2. Follow the first page of the attach email flow, click "add email address"
3. Notice you see the confirmation page correctly instead of only the username dialog
